backend/geonature/migrations/versions/829a376daa52_add_table_gn_synthese_t_reports.py

```diff
@@ -20,7 +20,7 @@ def upgrade():
     logger.info("Create t_reports table...")
     op.execute(
         """
-    CREATE TABLE gn_synthese.t_reports (
+    CREATE TABLE IF NOT EXISTS gn_synthese.t_reports (
```

Comment:
Idem, ne devrait pas être nécessaire.

Reply:
j'ai eu des soucis en faisant un update, je me suis retrouvé avec des doublons en base. Peut-être parce que j'avais fait un import à la mano avant, mais non nécessaire dans votre cas est-inutile pour tous ?

Reply:
IF NOT EXISTS
ON CONFLICT DO NOTHING

...Supprimés.